src/controlles/productController.js

```diff
@@ -0,0 +1,92 @@
+const { join } = require("path");
+const productService = require("../services/productService");
+const { json } = require("express");
+// 장바구니 상품 목록 호출
+const shoppingItems = async (req, res) => {
+    try {
+        const userInfo = req.user
+        const result = await productService.shoppingCart(userInfo);
+        return res.status(202).json({message: result});
+    } catch(error) {
+        return res.status(400).json({ message: "SHOPPINGITEMS ERROR", error });
+    }
+};
```

Comment:
```suggestion
const getShoppingItems = async (req, res) => {
    try {
        const userInfo = req.user
        const result = await productService.shoppingCart(userInfo);
        return res.status(202).json({message: result});
    } catch(error) {
        return res.status(400).json({ message: "SHOPPINGITEMS ERROR" });
    }
};
```
`error`를 프론트에게 그대로 return하면 백엔드의 에러를 프론트에게 그대로 전송하게 됩니다.
구체적인 이유가 없다면, 수행할 필요가 없는 동작입니다 :)